app.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const {getTopics,} = require('./controllers');
+
+
+const app = express();
+
+app.use(express.json());
+
+app.get("/api/topics",getTopics);
+
+
+
+
+
+
+
+app.listen(8080, () => {});
```

Comment:
please put this in a separate listen.js file. Supertest will attach its own custom listener to the express app.